src/components/LinkRenderer/LinkRenderer.tsx

```diff
@@ -1,17 +1,44 @@
 import { memo } from 'react';
+import s from './LinkRenderer.module.css';
 
-export const LinkRenderer = memo(function (props: any) {
+// types expected by react-markdown library
+type ElementWrapProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+>;
+
+export const LinkRenderer = memo(function (props: ElementWrapProps) {
   return (
-    <a
-      href={props.href}
-      target="_blank"
-      rel="noreferrer"
-      onClick={stopPropagation}
-    >
+    <a href={props.href} target="_blank" rel="noreferrer" onClick={stopPropagation}>
       {props.children}
     </a>
   );
 });
+
+// 192 and 12 are the values proposed by the task #8798
```

Comment:
We have task number in the commit message, so this comment seems not that useful.
I meant to point out that we slice it from the end of the string.

Reply:
can be handy if someone don't have git-lense extension